uiautomationtools/selenium/selenium_appium_shared.py

```diff
@@ -83,10 +81,10 @@ def find_element_explicitly(self, value, by='xpath', timeout=15, safe=False, man
                     if value != 'body':
                         if 'native' not in self.context.lower():
                             element = self.execute_script(
-                                "return arguments[0].filter(e => e.getAttribute('class'));", element)
+                                "return arguments[0].filter(e => e.getAttribute('attribute'));", element)
```

Comment:
i think u need to remove the quotes around attribute.

Reply:
Addressed